code/game/objects/effects/temporary_visuals/miscellaneous.dm

```diff
@@ -153,3 +153,10 @@
 	icon_state = "bhole3"
 	layer = ABOVE_LYING_MOB_LAYER
 	duration = WRAITH_BANISH_BASE_DURATION+1 //So we don't delete our contents early
+
+/obj/effect/temp_visual/order/temporary_supply_beacon
+	name = "temp supply beacon"
+	icon = 'icons/misc/mark.dmi'
+	icon_state_on = "x3"
+	layer = ABOVE_LYING_MOB_LAYER
```

Comment:
why this?

Reply:
the user of the antenna beacon might be confused to think that the supply is always on him, while no, when he uses the antenna it mark its current location. having a marker on the zone make the feature more intuitive